docs/user/whats-new.asciidoc

```diff
@@ -92,5 +92,17 @@ You can also now
 create embedded visualizations in your *Canvas* workpads.
 Just like in *Dashboard*, you can promote favorites to the *Visualization Library* for re-use.
 
+[float]
+=== {ml-cap} model management
+
+You can now add more models to bring modern 
+{ml-docs}/ml-nlp.html[{nlp} (NLP)] use cases to the {stack}. NLP models exist 
```

Comment:
The top-level page isn't too informative, so I'd recommend using the overview instead:
```suggestion
{ml-docs}/ml-nlp-overview.html[{nlp} (NLP)] use cases to the {stack}. NLP models exist 
```